### PR TITLE
sysdump: Collect new CR introduced as part of 1.13

### PR DESCRIPTION
**Pull request by sayboras**
This commit is to make sure that we have enough details for debugging
and troubleshooting.

### CODE REVIEW

**Review by nathanjsweet — APPROVED** [review]

File: sysdump/sysdump.go [hunk 1]
```diff
@@ -424,20 +424,6 @@ func (c *Collector) Run() error {
...
-				v, err := c.Client.ListCiliumEgressNATPolicies(ctx, metav1.ListOptions{})
-				if err != nil {
-					return fmt.Errorf("failed to collect Cilium egress NAT policies: %w", err)
-				}
-				if err := c.WriteYAML(ciliumEgressNATPoliciesFileName, v); err != nil {
-					return fmt.Errorf("failed to collect Cilium egress NAT policies: %w", err)
-				}
-				return nil
-			},
-		},
```
Comment (tklauser) [inline sysdump/sysdump.go:440]: I think we still want to collect `CiliumEgressNATPolicies` when running `cilium sysdump` against Cilium v1.12 or earlier.

**Review by tklauser — CHANGES_REQUESTED** [review]

File: sysdump/sysdump.go (hunk 1 above)
Comment (sayboras) [reply sysdump/sysdump.go:440]: the generated client was also removed from cilium v1.13.x, so if we want to to the above, we might need to use other kind of client (e.g. dynamic client), what do you think ?

Comment (tklauser) [reply sysdump/sysdump.go:440]: Sounds good. Or just copy the generated client to cilium-cli code base - whatever is easier to implement.

Comment (sayboras) [reply sysdump/sysdump.go:440]: thanks, let me give it a crack and get back to you.

**Review by tklauser — APPROVED** [review]

**Pull request merged**